viz/cmd/allow-scrapes.go

```diff
@@ -0,0 +1,108 @@
+package cmd
+
+import (
+	"os"
+
+	"github.com/spf13/cobra"
+)
+
+const (
+	allowScrapePolicy = `---
+apiVersion: policy.linkerd.io/v1beta1
+kind: Server
+metadata:
+  name: proxy-admin
+  labels:
+    linkerd.io/extension: viz
```

Comment:
@olix0r 
> Is it it intended that these resources do not include a namespace?

Yeah, I mentioned that in the PR description:
> We could consider taking a namespace as a CLI argument and templating the YAML so that it creates resources in that namespace, but I don't really think this has a benefit over just using kubectl apply -n <namespace> to apply the resources in the desired ns. This feels more unix-philosophy-y anyway...


@adleong
> The reason to mark them as part of the viz extension would be so that they can be found and deleted by `linkerd viz uninstall`.  However, this will require updating the viz uninstall command to do so.

I'll update this PR so that the uninstall command also deletes the policy resources.

Reply:
I do think the created-by annotation is a good idea.

Reply:
do we need to change this to a Helm template so that the `created-by` annotation can be set with the Linkerd version, then?

Reply:
You can get the version from https://github.com/linkerd/linkerd2/blob/main/pkg/version/version.go#L13 and insert it into the manifest with sprintf templating or go templating or helm templating or string replace or whatever is easiest.  I'd bias toward using the least powerful technique that gets the job done.